site-src/guides/metrics.md

```diff
@@ -116,6 +116,54 @@ kubectl -n default port-forward inference-gateway-ext-proc-pod-name  9090
 curl -H "Authorization: Bearer $TOKEN" localhost:9090/metrics
 ```
 
+## Setting Up Grafana + Prometheus
+
+### Grafana
+
+A simple grafana deployment can be done with the following commands:
+
+```bash
+     helm repo add grafana https://grafana.github.io/helm-charts
+     helm install grafana grafana/grafana --namespace monitoring
```

Comment:
I'm running through the guide using a fresh cluster. 
`monitoring` namespace doesn't exist and I get this error:
```bash
helm install grafana grafana/grafana --namespace monitoring
Error: INSTALLATION FAILED: create: failed to create: namespaces "monitoring" not found
```